groovy/postInit/chemistry/inorganic_chemistry/elements/d_block/group10/NickelChain.groovy

```diff
@@ -112,7 +112,7 @@ for (combustible in combustibles()) {
 
 ELECTROLYTIC_CELL.recipeBuilder()
         .fluidInputs(fluid('nickel_sulfate_solution') * 1000)
-        .fluidInputs(fluid('water') * 1000)
+        .fluidInputs(fluid('water') * 2000)
```

Comment:
does this change break stoik:

NiSO4.H2O + 2H2O > Ni + H2SO4.H2O + O

left side has 7 O, right side has 6 O